src/main/webapp/features/screens/game/dashboard/gameMap/ct-game-map.js

```diff
@@ -31,7 +43,29 @@ angular.module('catan')
                         var edgeId = angular.element(event.target).closest(DrawMapService.EDGE_SELECTOR).attr('edge-id');
                         SelectMapObjectService.select('edge', edgeId);
                     });
+
                 }
             };
 
+            //TODO: move that code to some helper?
+            function highlightHexes(element, map, dice, timeout) {
+                var hexesToHighlight = map.hexes.filter(function(hex) {
+                    return (dice === 7) ? hex.robbed : !hex.robbed && hex.dice === dice;
+                });
+
+                $timeout.cancel(timeout);
+                element.find(DrawMapService.HEX_SELECTOR).removeClass(HEXES_HIGHLIGHT_CLASS);
```

Comment:
If there was timer which was working for previous animation and we stop it without removing of HEXES_HIGHLIGHT_CLASS  class, then animated hexes will be still animated, but they shouldn't.

Actually, we can remove all this complex thing with timers, but then animation of different dice actions can be happening in the same time